src/docdb/tree/DocDBDatabaseTreeItemBase.ts

```diff
@@ -134,6 +134,19 @@ export abstract class DocDBDatabaseTreeItemBase extends DocDBTreeItemBase<Collec
         } catch (err) {
             return "Input must be a number"
         }
-        return null;
+        return undefined;
+    }
+
+    private static validateCollectionName(name: string): string | undefined | null {
+        if (!name) {
+            return "Name may not be empty";
```

Comment:
nit: You're not consistent with your error messages accross docdb and mongo. I would prefer "Collection name cannot" over "Name may not" or "Name cannot"